release_notes/ocp-4-17-release-notes.adoc

```diff
@@ -934,6 +934,16 @@ BMER was deprecated in {product-title} version 4.15 and 4.16. With this release,
 [id="ocp-4-17-image-registry-bug-fixes_{context}"]
 ==== Image Registry
 
+* In {product-title} 4.14, installing a cluster with Azure AD Workload Identity was made generally available. With that feature, administrators can configure a Microsoft Azure cluster to use Azure AD Workload Identity. With Azure AD Workload Identity, cluster components use temporary security credentials that are managed outside of the cluster. 
++
+Before this update, when {product-title} was deployed on Azure clusters with Azure AD Workload Identity, storage accounts that were created for the cluster and the image registry had *Storage Account Key Access* enabled by default. The use of shared access keys could create security risks for some {product-title} on Azure deployments.
++
+With this update, shared access keys are disabled by default on new installations that use Azure AD Workload Identity. This change enhances security by preventing the use of shared access keys.
++
+Existing storage accounts created before this update do not automatically have shared access keys disabled. Administrators must manually disable shared access keys support on their storage accounts to prevent the use of shared keys.
```

Comment:
Should we make it clear here as well that shared access keys can only be disabled if the cluster is configured to use workload identity? If customers disable shared access key on non-workload identity cluster the image registry operator will become degraded.

Reply:
done!